directory-menu@torchipeppo/files/directory-menu@torchipeppo/popup_menu.py

```diff
@@ -136,12 +137,17 @@ def launch(self, uri, timestamp):
             "uri": uri,
             "timestamp": timestamp,
         }))
+        subprocess.run(['/usr/bin/xdg-open', uri], check=False)
```

Comment:
[1/3] In my design for this version, the launch and open_terminal_at_path functions *are* supposed to do nothing but printing. This sends the JSON string on stdout to be received by applet.js, which then takes care of executing the requested action.

Reply:
Hmm. That wasn't working for me. I'll test this on another machine. I do like the gsettings lookup as a cleaner alternative that doesn't require a new preference.

Reply:
Did you test this with other terminals than gnome-terminal?

Reply:
My default terminal is `terminator` actually

Reply:
> My default terminal is `terminator` actually

Same! I'm a newcomer as of a few months. You have great taste! 😁